spec/ffi/struct_spec.rb

```diff
@@ -1026,4 +1026,58 @@ class BuggedStruct < FFI::Struct
     expect { expect(s[:data][1]).to == 0x12345678 }.to raise_error(IndexError)
   end
 end
+
+describe "Struct order" do
+  before :all do
+    @struct = Class.new(FFI::Struct) do
+      layout :value, :int32
+    end
+  end
+
+  before :each do
+    @pointer = @struct.new
+    @pointer.pointer.write_string("\x1\x2\x3\x4")
```

Comment:
I think this should be `write_bytes` since `write_string` [might append](https://github.com/ffi/ffi/pull/806) a final \0 byte in future ffi releases.

Reply:
Done.